test/integration/functional_test.go

```diff
@@ -1663,6 +1664,29 @@ func validateCertSync(ctx context.Context, t *testing.T, profile string) {
 	}
 }
 
+// validateNotActiveRuntimeDisabled asserts that for a given runtime, the other runtimes disabled, for example for containerd runtime, docker and crio needs to be not running
+func validateNotActiveRuntimeDisabled(ctx context.Context, t *testing.T, profile string) {
+	disableMap := map[string][]string{
+		"docker":     {"crio"},
+		"containerd": {"docker", "crio"},
+		"crio":       {"docker", "containerd"},
+	}
+
+	expectDisable := disableMap[ContainerRuntime()]
+	for _, cr := range expectDisable {
+		// for example: minikube sudo systemctl is-active docker
+		rr, err := Run(t, exec.CommandContext(ctx, Target(), "-p", profile, "ssh", fmt.Sprintf("sudo systemctl is-active %s", cr)))
+		if err != nil {
+			t.Logf("output of %s: %v", rr.Output(), err)
```

Comment:
t.Fatalf() here

Reply:
this doesn't need to be Error or Fatal, since when the container runtime is inactive it is technically an error, but it should be an error only if it is inactive when it is not supposed to be one

Reply:
does `sudo systemctl is-active` return and error if a service is stopped? Doesn't and error here mean `systemctl` itself is broken?

Reply:
just checked. It doesn't :)